GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift

```diff
@@ -5,7 +5,7 @@
 import UIKit
 
 extension SecureConversations {
-    final class FilePreviewView: UIView {
+    final class FilePreviewView: BaseView {
```

Comment:
I did it because previously adding subviews and establishing contains happened in the initializer is not the fully correct way, as a result of it SnapshotTest generated a white screen instead of all layout. I thought about moving establishing constraints into updateConstratint methods, however, decided to add consistency also and:
- changed base class to BaseView
- moved addSubview to setup
- moved constraints stuff to defineLayout
- `makeView` sets translatesAutoresizingMaskIntoConstraints to false, however, it doesn't required if it placed in stack view.

If you think these changes are not necessary for now and better to fix issue only without additional changes - let me know and I will revert changes back.

Reply:
Got it, so partially it was necessary for making snapshots work. Thanks for explanation. I don't mind these changes. We have FileUpload related stuff covered in chat screen so probably it is safe to proceed as is.